study.md

```diff
@@ -24,7 +24,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, how Ember "stops breaking the web".
 
 ```md
-<!-- your answer here -->
+Javascript-based SPAs broke the web by putting entire web apps within a single URL.  As a result, people are unable to share specific portions of content or use the back button.  Ember's routing and urls solve this problem by setting content under specific routes / urls, allowing people to share specific content (or states) in apps instead of the whole thing.  Ember saves states, allowing the back button to be used again.
```

Comment:
Yes.  URLs are important because data is important, and because we share information via url links.  URLs connect one piece of data to another, which is the basis of how the "web" is woven.